src/app/services/groupsocket/groupsocket.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { Subject } from 'rxjs/subject';
+import { Http } from '@angular/http';
+import * as io from 'socket.io-client';
+import 'rxjs/Rx';
+
+export interface UserInfo {
+  socketID : string;
+  groupUID : string;
+  username : string;
+  latitude : number;
+  longitude : number;
+
+}
+
+
+
+@Injectable()
+export class GroupSocketService {
+  //LOCALHOST
+  socketHost: string = 'http://localhost:8080';
+  //IP TO ACCESS WITH ANDROID EMULATOR. COMMENT OUT ALL OTHERS WHEN TESTING WITH ANDROID EMULATOR.
+  //socketHost: string = "http://10.0.2.2:8080";
+  socket: io;
+  uid: Observable<string>;
+  userInfos : Array<UserInfo> = [];
+<<<<<<< HEAD
+  userInfoSubject : Subject<UserInfo> = new Subject<UserInfo>();
+=======
+>>>>>>> ab72cdc0bd48779c886c5b43aacb992f9bbd2c2f
```

Comment:
It looks like on lines 28-31, 45-48, and 54-57, there's some merge conflicts.  Please resolve them. If you run `ionic serve`, it should complain about it when it's building the app.

Reply:
Resolved